pages.fr/common/cargo.md

```diff
@@ -0,0 +1,37 @@
+# cargo
+
+> Gestion d'un projet Rust et leur dependence (crates).
+> Certaines sous-commandes comme `cargo build` ont leurs propres documentations.
+> Plus d'informations: <https://crates.io/>.
```

Comment:
Shouldn't there be a space in front of all colons?
```suggestion
> Plus d'informations : <https://crates.io/>.
```

Reply:
> Shouldn't there be a space in front of all colons?

Yes, there must be space in front of all colons. Thanks for pointing it out.